pkg/operator/ceph/cluster/watcher.go

```diff
@@ -38,6 +39,8 @@ type clientCluster struct {
 	context   *clusterd.Context
 }
 
+var nodesCheckedForReconcile = util.NewSet()
```

Comment:
can we move this `var nodesCheckedForReconcile = util.NewSet()` down under `onK8sNode` method?

Reply:
I think not. The idea is to keep it global to use the same copy for each reconcile.